x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/10_basic.yml

```diff
@@ -333,3 +333,15 @@ setup:
   - match: {values.0: ["1",2.0,null,true,123,123]}
   - match: {values.1: ["1",2.0,null,true,123,123]}
   - match: {values.2: ["1",2.0,null,true,123,123]}
+
+---
+version is not allowed:
+  - requires:
+      cluster_features: ["gte_v8.14.0"]
```

Comment:
This is fine because we're forward porting this - 8.14 already has this change applied.

Reply:
:+1: